### ISSUE TITLE
I run it and shows  No such module 'BHBezelNotification'

### ISSUE THREAD

**Issue by jasonxxp** [BenLeggiero/BHBezelNotification-Demo-App#1]
I run it and shows  No such module 'BHBezelNotification'

**Comment by mattmartini** [comment]
Yes, I have the same problem.  How do you add the `BezelNotification` source code to this app?

**Comment by KyNorthstar** [comment]
Looking into it

**Comment by KyNorthstar** [comment]
@mattmartini @jasonxxp I've rewritten the demo app in SwiftUI, and linked it to the alpha release of the package rewrite.

Could you test it out and let me know if it works? You'll have to checkout the branch `feature/1-no-such-module-BHBezelNotification`.

See pull request #3, which I'll merge once let me know it works

**Comment by mattmartini** [comment]
Ky,

I did fresh clones of both repositories [BezelNotification-Demo-App](git@github.com:KyNorthstar/BezelNotification-Demo-App.git) and [BezelNotification](git@github.com:BlueHuskyStudios/BezelNotification.git) and did a `git fetch -all` and then a `git switch feature/1-no-such-module-BHBezelNotification` in each.  

Then using `Xcode` (26.4 on a MacStudio running Tahoe 26.4) I built `BezelNotification` and then `BezelNotification-Demo-App` The builds succeeded and I was able to run the demo app.  I added an image (twice, one of your images and one of mine) and tested the three timeouts (short, long, forever), the Tint feature, and the hide all feature (on each timeout length).  

Everything worked.  

I did however get a number of warnings and one error.  You can see them in the attached screenshots.  Let me know if you need further clarification or would like me to assist in further testing.

I am not a `Swift` programmer, more of `perl`/`bash`/`python`/`ruby` one.  How would I turn this into a command-line app that I could call from one of the above languages?
Something like `bluetoast --timeout short --tint #ff3300 --icon /path/to/image.png` and `bluetoast --hideall` 

Two, completely non-essential, features to add would be the ability to change the color of the text, and for a fade-in/fade-out of the toast.

Matt

Your Image.
<img width="1646" height="1200" alt="Image" src="https://github.com/user-attachments/assets/5ee7c8ba-dbc0-4951-bfe3-5d46653d8c86" />

My Image with tint.
<img width="944" height="760" alt="Image" src="https://github.com/user-attachments/assets/801f9154-ceff-494c-b5c9-b7a85e0f3e2e" />

Error.
<img width="1384" height="452" alt="Image" src="https://github.com/user-attachments/assets/9ce5b767-3090-4215-8b11-b8cbe7cd1a14" />

Warning. 
<img width="1391" height="888" alt="Image" src="https://github.com/user-attachments/assets/eccee002-0aad-42f3-b34f-a3738478b15d" />

Warning.
<img width="1385" height="887" alt="Image" src="https://github.com/user-attachments/assets/91a354ac-6bfa-4707-a39b-c544eacf3a99" />

Warning.
<img width="1391" height="889" alt="Image" src="https://github.com/user-attachments/assets/92075d84-82f2-454c-94f1-ed55d1eeef4d" />

Warning.
<img width="1384" height="452" alt="Image" src="https://github.com/user-attachments/assets/8a23c905-4798-4137-8bfa-abc24e5dfab9" />

Depreciation.
<img width="339" height="111" alt="Image" src="https://github.com/user-attachments/assets/54d50d99-1125-4e98-9683-7e78e2a8c1b7" />

**Comment by mattmartini** [comment]
I didn't notice it before, but I got this warning for BezelNotification.  All of the previous screenshots were from BezelNotification-Demo-App.

<img width="1388" height="755" alt="Image" src="https://github.com/user-attachments/assets/bb2bb510-7618-4da5-8d30-ed961a72f93a" />

**Comment by KyNorthstar** [comment]
Thank you so much, @mattmartini! I'll add these as subtasks before the PR is merged

**Comment by KyNorthstar** [comment]
> bluetoast --timeout short --tint #ff3300 --icon /path/to/image.png

That's an excellent idea. Would you like to [add that as a feature request](https://github.com/BlueHuskyStudios/BezelNotification/issues/new/choose) to BlueToast, or shall I?

**Comment by KyNorthstar** [comment]
> I didn't notice it before, but I got this warning for BezelNotification. All of the previous screenshots were from BezelNotification-Demo-App.

Added as https://github.com/BlueHuskyStudios/BezelNotification/issues/16

**Comment by KyNorthstar** [comment]
Alright, resolved all those warnings and made issues for the rest. It's mergin' time

**Comment by KyNorthstar** [comment]
@mattmartini @jasonxxp Version [3.0.0 Beta 1](https://github.com/KyNorthstar/BezelNotification-Demo-App/releases/tag/3.0.0-beta.1) should address all these issues

**Comment by mattmartini** [comment]
Ky,

I pulled the nightly tags (3.0.0 2026.04.27) for `BezelNotification` and `BezelNotification-Demo-App` and they compiled successfully.  I had to change my Signing Team to my personal certificate first.

I tried all of the options in the demo: `System Bezel`; `Toasts`: `Bezel`, `Snackbar`, `Capsule`.  Everything worked as expected.  

I went through each of the timeout options, changed text and images, tint, icon, ...   All were good.
The differences between the `Toast:Bezel:Effects` were subtle :-) 

I did get the "Support Ending for Intel-based Apps" warning for the demo app.

~~I could not see how to change to the `iOS` simulator so I only tested on `MacOS`.~~  See comment below

Let me know if you want anything specific tested.

**Comment by mattmartini** [comment]
Just saw that the `Toasts` Text support `Markdown`.  Nice.  

Looks like **bold**, *italic*, `code`, and ~~strikethru~~ are supported.  Headings, bullets, nor anything else seem to work.  What subset of `Markdown` is supported?

**Comment by mattmartini** [comment]
Figured out how to change the Product Destination and tested on `iPad (A16)` & `iPhone 17`.  

Everything tested ok.  Could not figure out how to use an icon, no images on the simulator.

Obviously there are no `System Bezel` on `iOS`

**Comment by KyNorthstar** [comment]
> @mattmartini 
> Just saw that the `Toasts` Text support `Markdown`. Nice.
> 
> Looks like **bold**, _italic_, `code`, and ~strikethru~ are supported. Headings, bullets, nor anything else seem to work. What subset of `Markdown` is supported?

Yes! In this demo app. Howl supports any `AttributedString` as the text of a toast. The demo uses the [`AttributedString.init(markdown:)`](https://developer.apple.com/documentation/foundation/instantiating-attributed-strings-with-markdown-syntax) initializer (without specifying any options, etc.), so it inherits the abilities and limitations of that.

**Comment by KyNorthstar** [comment]
Since everything seems good by you, I'll release 3.0.0 stable as soon as I finish wrapping it up

**Comment by mattmartini** [comment]
Fantastic.  I'm looking forward to the stable release and 3.1 😉
